examples/all-clusters-app/esp32/README.md

```diff
@@ -48,10 +48,20 @@ step. To install these components manually, follow these steps:
           $ cd ${HOME}/tools
           $ git clone https://github.com/espressif/esp-idf.git
           $ cd esp-idf
-          $ git checkout v4.4
+          $ git checkout release/v4.4
           $ git submodule update --init
           $ ./install.sh
 
+    To update an existing esp-idf toolchain to v4.4:
+
+          $ cd ~/tools/esp-idf
+          $ git fetch origin
+          $ git checkout release/v4.4
+          $ git reset --hard origin/release/v4.4
+          $ git submodule update --init
+          $ git clean -fdx
+          $ ./install.sh
```

Comment:
I think there should be a mention of `install.sh` and `export.sh` in this file. Can you please remove the other occurrence if it isn't clear